apps/root/src/constants/routes.tsx

```diff
@@ -35,7 +35,7 @@ export const EARN_ROUTE = {
   icon: <MoneyAddIcon />,
 };
 export const EARN_PORTFOLIO = {
-  label: defineMessage({ description: 'earnPortfolio', defaultMessage: 'Portfolio' }),
-  key: 'earnPortfolio',
+  label: defineMessage({ description: 'navigation.earn.portfolio', defaultMessage: 'Portfolio' }),
+  key: 'earn-portfolio',
```

Comment:
this should actually be /earn/portfolio right?, so to handle it like we do for dca

Reply:
Like it